guides-markdown/connect-local-nodes.md

```diff
@@ -0,0 +1,141 @@
+# Connect two local nodes
+
+```post-author
+Orfeas Stefanos Thyfronitis Litos
+```
+
+```post-description
+Start and manually connect two regtest nodes in a single script.
+```
+
+When we start two nodes in the regtest (loopback network, 127.0.0.1) with
+manually set ports, they don't get connected automatically because they don't
+know each other's port. We thus have to connect them manually. In this example
+we will fire up one SPV and one Full node and initiate a connection from the
```

Comment:
I think either "Full Node" or "full node". SPV is capitalized because it's an acronym, but "Full" isn't a proper noun on its own